torch/_inductor/fx_passes/post_grad.py

```diff
@@ -1131,6 +1150,10 @@ def make_dependencies_equivalent(
                 and node_device
                 and node_device.type == "cuda"
             ):
+                # cpu_indeg is a defaultdict.
+                # The line below is needed so as to create, if not existent, the entry in
+                # the dictionary. Then, we can delete it. Otherwise, we get a KeyError.
+                cpu_indeg[user]
```

Comment:
If we just keep this as a Counter we don't need these changes. can we go back to Counter ?